src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs

```diff
@@ -3154,6 +3264,13 @@ public new abstract class Arm64 : AdvSimd.Arm64
         public static unsafe ulong GetActiveElementCount(Vector<ulong> mask, Vector<ulong> from) => GetActiveElementCount(mask, from);
 
 
+        /// <summary>
+        /// svbool_t svrdffr()
+        ///   RDFFR Presult.B
+        /// </summary>
+        public static unsafe Vector<byte> GetFfr() => GetFfr();
```

Comment:
@tannergooding @terrajobst I think the APIs for `GetFfr` need another look. We can't provide overloads by return type, so if we were to expose a way to do that, we need to use suffix versions, i.e. `GetFfrByte` , `GetFfrUInt16`, etc.

Reply:
What you've done is actually how API review discussed and approved it, it just didn't get captured in the approval notes.

We have a standard convention of suffixing hwintrinsic APIs with the .NET type name where we need to disambiguate, such as `ConvertToSingle` or `LoadVector128`. This will always be a `Vector<T>` return and so we only needed to encode the `T`, such as `Byte`, `Int16`, `Int32`, etc.

Reply:
@TIHan - can you please update or add a comment to the original issue https://github.com/dotnet/runtime/issues/94004 the new names that we decided here? It will help us to know the final API name, rather than searching for this comment.

Reply:
Actually, thinking about it, we do not have individual type variants in C++ APIs. wondering why we have them at all? https://developer.arm.com/architectures/instruction-sets/intrinsics/#q=svrdffr. There should be just 1 API for this IMO - `Vector<byte> GetFfr()`

Reply:
> wondering why we have them at all?

C/C++ doesn't use templates and has an explicit mask register type (`svbool_t`). By default you'd presume that `svbool_t` is `byte`, but that isn't always the case and how that `svbool_t` is interpreted depends on the consuming operation. For example `CNTP` which finds the number of active elements set reads every `nth` bit:
```
if !HaveSVE() && !HaveSME() then UNDEFINED;
constant integer esize = 8 << UInt(size);
integer g = UInt(Pg);
integer n = UInt(Pn);
integer d = UInt(Rd);

CheckSVEEnabled();
constant integer VL = CurrentVL;
constant integer PL = VL DIV 8;
constant integer elements = VL DIV esize;
bits(PL) mask = P[g, PL];
bits(PL) operand = P[n, PL];
bits(64) sum = Zeros(64);

for e = 0 to elements-1
    if ActivePredicateElement(mask, e, esize) && ActivePredicateElement(operand, e, esize) then
        sum = sum + 1;
X[d, 64] = sum;

boolean ActivePredicateElement(bits(N) pred, integer e, integer esize)
    assert esize IN {8, 16, 32, 64, 128};
    integer n = e * (esize DIV 8);
    assert n >= 0 && n < N;
    return pred<n> == '1';
```

And so, a `svbool_t` for `int` and for `byte` are both the same size, which is `1/8th` the size of `Vector<T>` (thus for a 2048-bit Vector<T>, we have 256-bit `VectorMask<T>`). Where `VectorMask<byte>` means check every bit and `VectorMask<int>` means check every `4th` bit.

Since .NET uses generics and doesn't have an explicit mask type, we want to explicitly expose the underlying `T` so that devs can take the mask produced by `GetFfr` and pass it down to other APIs without casting, which is the same thing they could do in C/C++.